pkgs/desktops/plasma-5/powerdevil.nix

```diff
@@ -9,6 +9,13 @@
 mkDerivation {
   pname = "powerdevil";
   nativeBuildInputs = [ extra-cmake-modules kdoctools ];
+  patches = [
+    # Fix https://bugs.kde.org/show_bug.cgi?id=454161
+    (fetchpatch {
+      url = "https://invent.kde.org/plasma/powerdevil/-/merge_requests/94.patch";
```

Comment:
generally I get asked to use the specific commit rather than the PR that can change underneath us

```suggestion
      url = "https://invent.kde.org/plasma/powerdevil/-/commit/2ebe655d220c9167b66893a823b2fff2e2b8a531.patch";
```

Reply:
Agreed, though I'm hoping upstream will merge the change soon.

Reply:
If we want to merge this before upstream does then we must track a commit.

Reply:
Upstream merged, updated to point to the right commit.